lib/lightning_web/channels/run_with_options.ex

```diff
@@ -7,6 +7,12 @@ defmodule LightningWeb.AttemptJson do
   alias Lightning.Workflows.Job
   alias Lightning.Workflows.Trigger
 
+  @spec render(Attempt.t(), Keyword.t() | map()) :: map()
+  def render(%Attempt{} = attempt, options) do
```

Comment:
Can we add a type onto options please. It could arguably be a map at this point, but either way we should have some kind of safeguard as to the shape of the options we can pass to the worker.